remote/style.css

```diff
@@ -7,11 +7,11 @@ Description: Remote is a dark, minimal block theme ideal for bloggers. Its defau
 Requires at least: 5.8
 Tested up to: 5.9
 Requires PHP: 5.7
-Version: 0.0.13
+Version: 1.0.0
 License: GNU General Public License v2 or later
 License URI: https://raw.githubusercontent.com/Automattic/themes/trunk/LICENSE
 Text Domain: archeo
```

Comment:
It looks like this (Text Domain) and the Theme URI just need updating.

Reply:
good catch!